packages/dds/tree/src/core/forest/editableForest.ts

```diff
@@ -62,7 +62,7 @@ export function initializeForest(
 	if (visitAnchors) {
 		assert(forest.anchors.isEmpty(), 0x9b7 /* anchor set must be empty */);
 		const anchorVisitor = forest.anchors.acquireVisitor();
-		visitor = combineVisitors([visitor, anchorVisitor], [anchorVisitor]);
+		visitor = combineVisitors([visitor, anchorVisitor]);
```

Comment:
If we are going to stop invoking the AnnouncedVisitor methods, I'd expect to see AnnouncedVisitor removed and combineVisitors get updated. That would help prove we don't need any of this anymore.

Reply:
AnchorTreeIndex still creates an AnnouncedVisitor, we just don't need the AnchorSet to be an AnnouncedVisitor anymore. The forest allows AnnouncedVisitors to be registered and they get combined with the delta visitor and anchor visitor when acquireVisitor gets called on a forest.